docs/Changelog.md

```diff
@@ -0,0 +1,24 @@
+# Changelog
+
+>danger
+>API versions below v6 will be discontinued on October 16, 2017, after which they will be non-functioning.
+
+This document contains breaking changes between API versions.
+
+## Version 6
+
+* [Guild Channel](#DOCS_channel/channel-objects) Object
```

Comment:
there is not a concept of guild channel or dm channel.. for sake of simplicity you should merge these